conf/igenomes.config

```diff
@@ -89,6 +92,7 @@ params {
         'NCBI.GRCh38' {
             bwa                   = "${params.igenomes_base}/Homo_sapiens/NCBI/GRCh38/Sequence/BWAIndex/version0.6.0/"
             fasta                 = "${params.igenomes_base}/Homo_sapiens/NCBI/GRCh38/Sequence/WholeGenomeFasta/genome.fa"
+            ngscheckmate_bed      = "https://raw.githubusercontent.com/parklab/NGSCheckMate/master/SNP/SNP_GRCh38_hg38_wChr.bed"
```

Comment:
```suggestion
            ngscheckmate_bed      ="${params.igenomes_base}/Homo_sapiens/NCBI/GRCh38/Annotation/NGSCheckMate/SNP_GRCh38_hg38_wChr.bed"
```

Reply:
we should put these files on igenomes. What is the difference to the normal interval bed?

Reply:
there're on it already ;-)

Reply:
we fetched them on https://github.com/parklab/NGSCheckMate/tree/master/SNP